frontend/src/core/app/helpers.js

```diff
@@ -190,6 +186,14 @@ define(function(require){
 
         },
 
+        ifImageIsCourseAsset: function(url, block) {
```

Comment:
I'm not thrilled about the name myself but this function is a handlebars helper and the name is consistent with the other similar helpers in the same file, e.g. `ifValueEquals()`, `ifUserIsMe()`.  Handlebars doesn't support the ability to call a function with an `if` statement in front.